### PR TITLE
Feature/140 make deployment stage a pipeline template

### PR DESCRIPTION
**Pull request by paulo-teran-jalafund**
[AB#140](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/140) The new template works with both Project and Resumes pipelines

### CODE REVIEW

**Review by alejandro-castellon — DISMISSED** [review]

**Review by niver-copa-jalafund — DISMISSED** [review]

**Review by pedro-lima-jalafund — DISMISSED** [review]

File: azure-pipelines/netcore-deploy-template.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+parameters:
+  - name: 'azureSubscription'
+    default: 'devlevel3-dev (a797242d-ce60-41aa-98ca-2e22d5221cfe)'
```
Comment (fernando-maceda-jalafund) [inline azure-pipelines/netcore-deploy-template.yml]: not really sure if we should have this subscription data here, maybe @martin-callejas-jalafund can confirm?

**Review by fernando-maceda-jalafund — COMMENTED** [review]
@martin-callejas-jalafund I would appreciate your review here, I know you're more familiar with it.

File: azure-pipelines/netcore-deploy-template.yml (hunk 1 above)
Comment (martin-callejas-jalafund) [reply azure-pipelines/netcore-deploy-template.yml]: yeah, my bad here bc it's been previously merged

@paulo-teran-jalafund please replace with this default value: `devlevel3-dev-conn`

**Review by martin-callejas-jalafund — CHANGES_REQUESTED** [review]
that minor change needed, good job man

**Comment by paulo-teran-jalafund** [comment]
@martin-callejas-jalafund @fernando-maceda-fundacion  made the change, still working fine with both pipelines

**Review by niver-copa-jalafund — APPROVED** [review]

**Review by fernando-maceda-jalafund — APPROVED** [review]
LGTM

**Review by pedro-lima-jalafund — APPROVED** [review]

**Review by martin-callejas-jalafund — APPROVED** [review]

**Pull request merged**